lib/screen/irl_screen/webview_irl_screen.dart

```diff
@@ -90,27 +94,42 @@ class _IRLWebScreenState extends State<IRLWebScreen> {
       final addresses = await injector<CloudDatabase>()
           .addressDao
           .getAddressesByType(cryptoType.source);
+      final personas = await injector<CloudDatabase>().personaDao.getPersonas();
+      final listUuid = personas.map((e) => e.uuid).toList();
+      addresses.removeWhere((e) => !listUuid.contains(e.uuid));
       if (addresses.isEmpty) {
-        final persona =
-            await injector<AccountService>().getOrCreateDefaultPersona();
-        final addedAddress = await persona.insertNextAddress(
+        final personas =
+            await injector<CloudDatabase>().personaDao.getDefaultPersonas();
+        if (personas.isEmpty) {
+          return _logAndReturnJSResult(
+            '_countAddress',
+            JSResult.error('Address not found'),
+          );
+        }
+        final addedAddress = await personas.first.insertNextAddress(
             cryptoType == CryptoType.XTZ
                 ? WalletType.Tezos
                 : WalletType.Ethereum);
         addresses.add(addedAddress.first);
       }
       String? address;
-      if (addresses.length == 1) {
+      final params = arguments['params'] as Map?;
+      final minimumCryptoBalance =
+          int.tryParse(params?['minimumCryptoBalance'] ?? '') ?? 0;
+      if (addresses.length == 1 && minimumCryptoBalance > 0) {
```

Comment:
why we have to check `minimumCryptoBalance > 0`?

Reply:
yead, it should be == 0